content/en/serverless/guide/troubleshoot_serverless_monitoring.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Troubleshoot Serverless Monitoring
+kind: documentation
+further_reading:
+- link: '/serverless/installation/'
+  tag: 'Documentation'
+  text: 'Installing Serverless Monitoring'
+- link: '/serverless/guide/'
+  tag: 'Documentation'
+  text: 'Troubleshoot Common Issues'
+---
+
+This guide helps you troubleshoot general data collection issues for serverless monitoring, such as missing metrics, traces, logs or tags. Check out other [troubleshooting guides][1] for specific and common issues, such as function code size is too large, and webpack compatibility.
+
+## Understand the basics
+
+To better understand the instructions from this guide, be sure to familiarize yourself with the [key concepts][2] first. A better understanding on how things work helps you identify the missing pieces and narrow down probable causes.
+
+## Use Datadog Lambda Extension instead of the Forwarder
+
+If you are still using the [Datadog Forwarder Lambda function][3] for data collection, consider adopting the [Datadog Lambda Extension][4] instead. Many known issues due to the technical limitations from the Forwarder Lambda can be resolved automatically after migrating to the Lambda Extension.
```

Comment:
```suggestion
If you are still using the [Datadog Forwarder Lambda function][3] for data collection, consider adopting the [Datadog Lambda Extension][4] instead. Because the technical limitations of the Forwarder Lambda cause a number of known issues, migrating to the Lambda Extension can resolve them automatically.
```